temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package temporal.api.cloud.namespace.v1;
+
+option go_package = "go.temporal.io/api/cloud/namespace/v1;namespace";
+
+import "temporal/api/cloud/sink/v1/message.proto";
+
+import "google/protobuf/timestamp.proto";
+
+message CertificateFilterSpec {
+    // The common_name in the certificate
+    string common_name = 1;
+    // The organization in the certificate
+    string organization = 2;
+    // The organizational_unit in the certificate
+    string organizational_unit = 3;
+    // The subject_alternative_name in the certificate
+    string subject_alternative_name = 4;
+}
+
+message CodecServerPropertySpec {
+    // Server endpoints
+    string endpoint = 1;
+    // Whether to pass access token, i.e. jwt
+    bool pass_access_token = 2;
+    // Whether to include credentials
+    bool include_credentials = 3;
+}
+
+message NamespaceSpec {
+    // The region where the namespace is (or will be) active
+    string region = 1;
+    // The base64 encoded ca cert(s) that the clients can use for authentication and authorization
+    string accepted_client_ca = 2;
+    // The number of days the workflows data will be retained for
+    int32 retention_days = 3;
+    // The custom search attributes to use for the namespace
+    // The name of the attribute is the key and the type is the value
+    // Supported attribute types: text, keyword, int, double, bool, datetime, keyword_list
+    map<string, string> search_attributes = 4;
+    // Certificate filters which, if specified, only allow connections from client certificates
+    // whose distinguished name properties match at least one of the filters
+    repeated CertificateFilterSpec certificate_filters = 5;
+    // Environment of the namespace. - optional
+    // NOTE: currently there is no additional SLA or functional guarantee implied by the value of this field.
+    // supported environments: dev, test, prod
+    string environment = 6;
+    // Codec server property spec needed for user to set and retrieve - optional
+    CodecServerPropertySpec codec_spec = 7;
+    // The regions where the namespace is (or will be) located - optional
+    repeated string passive_regions = 8;
+    // The export sink specifications keyed on the sink name - optional
+    map<string, temporal.api.cloud.sink.v1.ExportSinkSpec> export_sinks = 9;
```

Comment:
Arguably "export" is too generic of a word for this. Is this just history export? Or is this also log export, audit export, user export, and all other exports we may support henceforth?